packages/graphback-codegen-resolvers/src/templates/LayeredResolverTemplates.ts

```diff
@@ -72,7 +72,7 @@ export const typeRelation = (relation: string, columnName: string, fieldName: st
   if (relation === 'OneToOne') {
     return `${fieldName}: (parent, args, context) => {
       validateRuntimeContext(context)
-      return context.crudService.batchLoadData("${tableName}", "${columnName}", parent.id, context)
+      return context.crudService.findBy("${tableName}", { id: parent.${columnName} })
```

Comment:
[Requires Discussion] Missing many2many. What will be our plan for supporting many2many relationship and querying.

Reply:
Logging issue for this as I feel that it requires separate discussion.